docs/reference/microsoft_teams.md

```diff
@@ -0,0 +1,157 @@
+
+## Availability and prerequisites
+
+This connector is available as a **connector client** from the **Python connectors framework**. To use this connector, satisfy all [connector client requirements](https://www.elastic.co/guide/en/enterprise-search/master/build-connector.html).
+
+This connector is in **beta** and is subject to change. The design and code is less mature than official GA features and is being provided as-is with no warranties. Beta features are not subject to the support SLA of official GA features.
+
+## Usage
+
+To use this connector as a **connector client**, use the **Microsoft Teams** tile from the connectors list **Customized connector** workflow.
+
+For additional operations, see [Usage](https://www.elastic.co/guide/en/enterprise-search/master/connectors-usage.html).
+
+## Connecting to Microsoft Teams
+
+To connect to Microsoft Teams you need to [create an Azure Active Directory application and service principal](https://learn.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal) that can access resources. Follow these steps:
+
+1. Go to the Azure portal (https://portal.azure.com) and sign in with your Azure account.
+
+2. Navigate to the "Azure Active Directory" service.
+
+3. Select "App registrations" from the left-hand menu.	
+
+4. Click on the "New registration" button to register a new application.
+
+5. Provide a name for your app, and optionally select the supported account types (e.g., single tenant, multi-tenant).
+
+6. Click on the "Register" button to create the app registration.
+
+7. After the registration is complete, you will be redirected to the app's overview page. Take note of the "Application (client) ID" value, as you'll need it later.
+
+8. Scroll down to the "API permissions" section and click on the "Add a permission" button.
+
+9. In the "Request API permissions" pane, select "Microsoft Graph" as the API.
+
+10. Choose the permissions and select the following permissions:
+    - User.Read.All (Delegated and Application)
+    - TeamMember.Read.All (Delegated)
+    - Team.ReadBasic.All (Delegated)
+    - TeamsTab.Read.All (Delegated)
+    - Group.Read.All (Delegated)
+    - ChannelMessage.Read.All (Delegated)
+    - Chat.Read (Delegated) & Chat.Read.All (Application)
+    - Chat.ReadBasic (Delegated) & Chat.ReadBasic.All (Application)
+    - Files.Read.All (Delegated and Application)
+    - Calendars.Read (Delegated and Application)
+
+
+11. Click on the "Add permissions" button to add the selected permissions to your app.
+
+Finally, click on the "Grant admin consent" button to grant the required permissions to the app. This step requires administrative privileges. ℹ️ **NOTE**: If you are not an admin, you need to request the Admin to grant consent via their Azure Portal.
```

Comment:
Nit: `Finally` is used here but this isn't the final step, also `admin` is inconsistently capitalized.

```suggestion
Click on the "Grant admin consent" button to grant the required permissions to the app. This step requires administrative privileges. ℹ️ **NOTE**: If you are not an admin, you need to request the admin to grant consent via their Azure Portal.
```